### PR TITLE
FRW-6368 Integration

### PR DESCRIPTION
**Pull request by dimitriyTsemma**
Tickets:

Bugs:

- https://spryker.atlassian.net/browse/FRW-6369
- https://spryker.atlassian.net/browse/FRW-6368
- https://spryker.atlassian.net/browse/FRW-6378
- https://spryker.atlassian.net/browse/FRW-6474
- https://spryker.atlassian.net/browse/FRW-6362
- https://spryker.atlassian.net/browse/FRW-6365
- https://spryker.atlassian.net/browse/FRW-6376
- https://spryker.atlassian.net/browse/FRW-1804
- https://spryker.atlassian.net/browse/FRW-7280
- https://spryker.atlassian.net/browse/FRW-7311
- https://spryker.atlassian.net/browse/FRW-7313
- https://spryker.atlassian.net/browse/FRW-6942

Epic:
- https://spryker.atlassian.net/browse/FRW-6430 (PATCH, POST, PUT)

### CODE REVIEW

**Review by yaroslav-spryker — APPROVED** [review]

File: data/import/common/US/price_product_merchant_relationship.csv [hunk 1]
```diff
@@ -1,942 +1,942 @@
 merchant_relation_key,abstract_sku,concrete_sku,price_type,store,currency,price_net,price_gross
```
Comment (geega) [inline data/import/common/US/price_product_merchant_relationship.csv:1]: Currency column?

**Review by geega — APPROVED** [review]

File: data/import/common/common/glossary.csv [hunk 2]
```diff
@@ -1343,6 +1345,8 @@ product.filter.price-default-eur-net_mode,Preis,de_DE
 product.filter.price-default-eur-net_mode,Price range,en_US
 product.filter.price-default-eur-gross_mode,Preis,de_DE
 product.filter.price-default-eur-gross_mode,Price range,en_US
+product.filter.price-default-usd-net_mode,Price range,en_US
+product.filter.price-default-usd-gross_mode,Price range,en_US
```
Comment (asmarovydlo) [inline data/import/common/common/glossary.csv:1349]: Don't we need a DE translation?

File: data/import/common/common/cms_block.csv [hunk 3]
```diff
@@ -35,8 +35,8 @@ cms-block-email--availability-notification-unsubscribed--html,availability-notif
...
...    <tr>                           <td class=""sprykerTextContent"" colspan=""2"" style=""padding-top: 0px;padding-right: 0px;padding-bottom: 9px;padding-left: 9px;font-family:Helvetica, Arial, Verdana, sans-serif;font-size: 15px;font-weight: normal;text-align: left;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;"" valign=""top"">                             <!--{% for option in item.productOptions %}-->                               <p style=""color:#8F8F8F"">+ {{ option.groupName }}: {{ option.sumPrice | money(true, mail.order.currencyIsoCode) }}</p>                             <!--{% endfor %}-->                           </td>                         </tr>                       </tbody>                     </table>                   </td>                 </tr>               <!--{% endfor %}-->             <!--{% endfor %}-->           </tbody>         </table>         <!--[if mso]>         </td>         <![endif]-->         <!--[if mso]>         </tr>         </table>         <![endif]-->       </td>     </tr>   </tbody> </table> <!--[if (gte mso 9)|(IE)]> </td> </tr> </table> <![endif]--> <!-- // END TEMPLATE -->"
```
Comment (asmarovydlo) [inline data/import/common/common/cms_block.csv:38]: Only DynamicStore-related variables must be changed, but, here a lot more stuff changed. (At least image URLs)

Please fix.

File: data/import/common/US/currency_store.csv [hunk 4]
```diff
@@ -1,2 +1,3 @@
 currency_code,store_name,is_default
-USD,US,1
+EUR,US,1
+CHF,US,0
```
Comment (asmarovydlo) [inline data/import/common/US/currency_store.csv:3]: This and **all** similar changes, the bug that is integrated was about replacing EUR to USD for the US store, but, here we already have USD, why are we integrating smth?

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: data/import/common/common/glossary.csv (hunk 2 above)
Comment (dimitriyTsemma) [reply data/import/common/common/glossary.csv:1349]: this is just an integration
if we need it, then it should be inside the main PR
https://github.com/spryker/suite-nonsplit/pull/8613/files#diff-591493093b412548fb171d04b9e15b0b4c2543abdd5b6a41948c480b94a6db6c

File: data/import/common/US/currency_store.csv (hunk 4 above)
Comment (dimitriyTsemma) [reply data/import/common/US/currency_store.csv:3]: please check https://github.com/spryker/suite-nonsplit/pull/8647

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
All the currency-related changes that @vol4onok did in his PR related to the https://github.com/spryker/suite-nonsplit/blob/ee78bdaf28b2f0f52b42255aa430c6e70d983bc5/config/Shared/stores.php#L84 and the currencies that are there. 

Here, in demoshop, we have only USD currency, so we don't need those changes at all. Please get rid of them.

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**